packages/backend/routes/v1/crm/utils/close.ts

```diff
@@ -0,0 +1,121 @@
+import axios from 'axios';
```

Comment:
@gold-olar could we please call this folder `authHandlers` or something similar? I think if we call it `utils` it might just get misused.

Reply:
got it.. would change that...